spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResourceBucket.java

```diff
@@ -110,11 +125,16 @@ public long lastModified() {
 	}
 
 	@Override
-	public Resource createRelative(String relativePath) throws IOException {
-		return new GoogleStorageResourceObject(this.storage,
+	public GoogleStorageResourceObject createRelative(String relativePath) throws IOException {
```

Comment:
This method definitely needs a javadoc even though it overrides because the behavior is pretty nuanced.